pkg/patterns/declarative/pkg/applier/direct.go

```diff
@@ -27,24 +26,36 @@ func (d *DirectApplier) Apply(ctx context.Context,
 	validate bool,
 	extraArgs ...string,
 ) error {
+	tmpFile, err := ioutil.TempFile("", "tmp-manifest")
```

Comment:
Do you think it'll be safer to have some randomness in the name?
I've a use case where two go routines use the applier on separate manifests at the same time.
With a constant name, there can be conflict.
A random name with a prefix and/or suffic may be safer?
I'm referring to the suffix example in https://golang.org/pkg/io/ioutil/#TempFile